apps/nextjs-website/react-components/types/Video/Video.types.ts

```diff
@@ -0,0 +1,47 @@
+import { CommonProps, Generic } from '../common/Common.types';
+
+export interface VideoImageProps extends CommonProps {
+  /** Video url */
+  src?:
+    | string
+    | {
+        alternativeText: string | null;
+        url: string;
+        mime: string;
+      };
```

Comment:
```suggestion
  src?: {
        url: string;
        mime: string;
      };
```
Better to just give one option (the one that looks more like Strapi).
Also, `alternativeText` was unused.